docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+API Reference
+=============
+
+Module contents
+---------------
+
+.. automodule:: scrapinghub.client
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Submodules
+----------
+
+activity
+--------
+
+.. automodule:: scrapinghub.client.activity
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
There's a problem here, showing inherited members is fine, but it doesn't cover methods that we set in runtime when instantiating an object with `_proxy_methods` logic, i.e. `iter` [method](https://github.com/scrapinghub/python-scrapinghub/blob/master/scrapinghub/client/activity.py#L48) for `Activitity`.